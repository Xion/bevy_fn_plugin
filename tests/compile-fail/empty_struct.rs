#[macro_use] extern crate bevy_fn_plugin;

#[bevy_plugin]
struct EmptyStruct;

fn main() {}
