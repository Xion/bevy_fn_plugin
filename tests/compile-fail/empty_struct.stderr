error: expected `fn`
 --> tests/compile-fail/empty_struct.rs:4:1
  |
4 | struct EmptyStruct;
  | ^^^^^^
